cloudferrylib/os/compute/nova_compute.py

```diff
@@ -658,3 +669,19 @@ def get_free_disk(self):
 
     def delete_vm_by_id(self, vm_id):
         self.nova_client.servers.delete(vm_id)
+
+
+def down_hosts(nova_client_):
```

Comment:
Why we make this and the next one functions as functions, but not as methods of the `NovaCompute` class?